packages/lib/src/vis/shared/Html.tsx

```diff
@@ -114,7 +112,10 @@ function Html(props: Props) {
 
     ReactDOM.render(
       <div
-        ref={innerRef}
+        ref={(elem) => {
+          refCallback?.(elem);
+          innerRef.current = elem || undefined;
```

Comment:
We already have an internal ref for this element, so here I'm passing a callback to `ref` that sets both the external ref and the internal ref.